hack/unwanted-dependencies.json

```diff
@@ -1,12 +1,7 @@
 {
   "spec": {
     "unwantedModules": {
-      "cloud.google.com/go": "cloud dependency",
```

Comment:
/hold
for unexpected update in `hack/unwanted-dependencies.json`
/lgtm cancel

Reply:
Done